src/Core/View.js

```diff
@@ -79,6 +82,44 @@ function View(crs, viewerDiv, options = {}) {
     if (__DEBUG__) {
         this.isDebugMode = true;
     }
+
+    if (options.loadingScreen !== false) {
+        let loadingScreenContainer;
+        if (!options.loadingScreen || options.loadingScreen === 'itowns') {
+            const node = document.createElement('style');
+            node.innerHTML = LoadingScreenCSS;
+            document.body.appendChild(node);
+            // loading screen
+            viewerDiv.insertAdjacentHTML('beforeend', LoadingScreenHTML);
+            loadingScreenContainer = document.getElementById('itowns-loader');
+        } else if (typeof (options.loadingScreen) === 'string') {
+            loadingScreenContainer = document.getElementById(options.loadingScreen);
+        } else if (options.loadingScreen instanceof (HTMLElement)) {
+            loadingScreenContainer = options.loadingScreen;
+        } else {
+            throw new Error('Invalid value for options.loadingScreen. Expected: false, string or HTMLElement');
+        }
+
+        const view = this;
+        // auto-hide in 3 sec or if view is loaded
+        const hideLoader = () => {
+            if (!loadingScreenContainer) {
```

Comment:
This `if` should probably be outside the fat-arrow function, to avoid setting it up when not needed.

The code never sets `loadingScreenContainer` to a falsy value (or any other value than it's initial one actually) so this `if` is useless. The code that calls `removeChild` should `null` the variable for that test to actually be useful (and even then, it wouldn't prevent the `hideLoader` body to be executed twice under some circumstances).